.gitignore

```diff
@@ -2,3 +2,5 @@
 Gemfile.lock
 .bundle
 vendor
+.idea
+lib/com
```

Comment:
@talevy `rake install_jars` (from jar-dependencies) brings in jars to `lib/com`. We may have to change `JARS_HOME` to `vendor`